index.js

```diff
@@ -92,20 +93,22 @@ document.querySelectorAll('.button-4').forEach((n, index) => {
 });
 
 // closing popup menu
-document.querySelector('#close-btn').addEventListener('click', () => {
-  overlay1.style.display = 'none';
+document.querySelector('#cancel').addEventListener('click', () => {
+  overlay.style.display = 'none';
 });
-
+// checking validation for email
 document.querySelector('.submit-b').onclick = function () {
-  console.log(email.innerText);
-  if (email.innerText !== email.innerText.toLowerCase()) {
+  if (email.value !== email.value.toLowerCase()) {
     overlay1.style.display = 'block';
+  } else {
+    overlay2.style.display = 'block';
   }
 };
-
-document.querySelector('#cancel').addEventListener('click', () => {
-  overlay.style.display = 'none';
-});
+// closing menu
+document.querySnelectorAll('#close-btn').forEach((n) => n.addEventListener('click', () => {
+  overlay2.style.display = 'none';
```

Comment:
![Screenshot from 2022-11-17 15-08-12](https://user-images.githubusercontent.com/65129228/202443060-6c3050fe-3bdc-4822-b994-10e590c66687.png)

- [ ] Great job implementing the validation pop-up message:100: . Kindly fix the method name for selecting a query instead of querySnelectorAll() and change it to querySelectorAll() for the close button functionality to work.

Example
```
const close-btn = document.querySelectorAll('#close-btn');
```